annotation/tests/test_tasks_services.py

```diff
@@ -0,0 +1,158 @@
+from unittest.mock import Mock, call, patch
+
+from annotation.models import File
+import pytest
+from sqlalchemy.orm import Session
+
+from annotation.errors import FieldConstraintError
+from annotation.jobs.services import ValidationSchema
+from annotation.tasks.services import validate_files_info, validate_task_info, validate_users_info
+
+
+TEST_PARAMS_INVALID_FILES = [
```

Comment:
Move this constant into appropriate test, no need for it here

Reply:
Should I use this as a parameterized test without the constant, or should the constant be placed directly before the function that uses it? I went with the latter approach.

Reply:
Fixed;)